### PR TITLE
Use rcpputils::find_and_replace instead of std::regex_replace

### PR DESCRIPTION
**Pull request by jacobperron**
The rcpputils implementation is more performant.

Relates to https://github.com/ros2/rmw_fastrtps/issues/277
Requires https://github.com/ros2/rcpputils/pull/13

### CODE REVIEW

**Comment by jacobperron** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/7350/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/3515/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/6028/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/7172/)

**Comment by jacobperron** [comment]
After running CI against master (https://github.com/ros2/rmw_opensplice/issues/264#issuecomment-506459958), it looks like the test failures already exist and are related to builds with OpenSplice.

File: rmw_opensplice_cpp/CMakeLists.txt [hunk 1]
```diff
@@ -28,12 +28,14 @@ if(NOT rosidl_typesupport_opensplice_cpp_FOUND OR NOT rosidl_typesupport_openspl
...
 endif()
 
+find_package(rcpputils REQUIRED)
 find_package(rcutils REQUIRED)
 find_package(rmw REQUIRED)
 find_package(rosidl_generator_c REQUIRED)
 find_package(rosidl_generator_cpp REQUIRED)
 
 ament_export_dependencies(
+  rcpputils
```
Comment (dirk-thomas) [inline rmw_opensplice_cpp/CMakeLists.txt]: @jacobperron The package exports the `rcpputils` dependency but doesn't declare that run dependency in the manifest.

Comment (jacobperron) [reply rmw_opensplice_cpp/CMakeLists.txt]: Addressed in https://github.com/ros2/rmw_opensplice/pull/277

**Pull request merged**